install_config/aggregate_logging.adoc

```diff
@@ -336,17 +342,21 @@ With parameters:
 ifdef::openshift-origin[]
 ----
 $ oc new-app logging-deployer-template \
-             --param IMAGE_VERSION=v1.2.0 \
+             --param IMAGE_VERSION=<tag> \
              --param MODE=install
 ----
 endif::openshift-origin[]
 ifdef::openshift-enterprise[]
 ----
 $ oc new-app logging-deployer-template \
-             --param IMAGE_VERSION=3.3.0 \
+             --param IMAGE_VERSION=<tag> \
              --param MODE=install
 ----
+
 endif::openshift-enterprise[]
```

Comment:
Now that's there's no actual difference between these code blocks (thanks to the use of `<tag>`), you can remove the ifdefs here and just have a single code block / command, I think.